component/core/src/main/java/org/exoplatform/social/core/service/LinkProvider.java

```diff
@@ -201,7 +201,7 @@ public static String getProfileLink(String username, String portalOwner) {
                .append(identity.getProfile().getAvatarUrl())
                .append("',")
                .append("position: '")
-               .append(identity.getProfile().getPosition() == null ? "" : identity.getProfile().getPosition().replace("'", "\\\\'"))
+               .append(identity.getProfile().getPosition() == null ? "" : identity.getProfile().getPosition().replace("'", "\\\\'").replace("\"", "\\\\\""))
```

Comment:
Can you please do the same for the `FullName`

Reply:
done!